commit 5f665800fdbd42846bc7ee83a9fd356c59babb8f (refs/changes/82/3082/2)
Author: Trent Mick <trentm@gmail.com>
Date:   2017-12-08T17:01:14-08:00 (1 year, 10 months ago)
    
    RELENG-782 dhcpd build failure due to broken old SmartOS tar

diff --git a/Makefile b/Makefile
index 981d313..e14fdeb 100644
--- a/Makefile
+++ b/Makefile
@@ -130,7 +130,7 @@ pkg: all
 release: $(RELEASE_TARBALL)
 
 $(RELEASE_TARBALL): pkg
-	(cd $(PKG_DIR); tar -jcf $(TOP)/$(RELEASE_TARBALL) root)
+	(cd $(PKG_DIR); $(TAR) -jcf $(TOP)/$(RELEASE_TARBALL) root)
 
 publish:
 	@if [[ -z "$(BITS_DIR)" ]]; then \
